LoopKitHostedTests/HKAnchoredObjectQueryMock.swift

```diff
@@ -13,11 +13,21 @@ class HKAnchoredObjectQueryMock: HKAnchoredObjectQuery {
     let anchor: HKQueryAnchor?
     let resultsHandler: (HKAnchoredObjectQuery, [HKSample]?, [HKDeletedObject]?, HKQueryAnchor?, Error?) -> Void
     
+    @available(iOS 15.0, *)
```

Comment:
[nit] Not sure what this has to do with this PR. Consider creating a new "iOS 15 support" Jira ticket and moving this there.

Reply:
This is necessary in order for unit tests to pass in Xcode 13 on iOS 15.